src/components/api.js

```diff
@@ -8,4 +8,10 @@ const getAllArticles = () => {
   });
 };
 
-export { getAllArticles };
+const getSingleArticle = (article_id) => {
+  return api.get(`/api/articles/${article_id}`).then((response) => {
+    return response.data.article;
+  });
+};
+
```

Comment:
I like that you've used this function to get all the info off the relevant keys, keeps our components a bit neater 🧹